app/Kings_fighting/kings_start_position.py

```diff
@@ -0,0 +1,19 @@
+from __future__ import annotations
+from app.Kings_fighting.addition_tools import magic_upgrades
+
+
+class King:
```

Comment:
Why King if we have knights fighting?) I mean it's up to you, just seem not intuitive, as for me.